lib/Sema/CSSimplify.cpp

```diff
@@ -4826,6 +4826,27 @@ bool ConstraintSystem::repairFailures(
   case ConstraintLocator::ApplyArgToParam: {
     auto loc = getConstraintLocator(locator);
 
+    // If this type mismatch is associated with a synthesized argument,
+    // let's just ignore it because the main problem is the absence of
+    // the argument.
+    if (auto applyLoc = elt.getAs<LocatorPathElt::ApplyArgToParam>()) {
```

Comment:
@LucianoPAlmeida not sure what happened with your comment but it’s not always true because `LValueConversion` is a fallthrough here.

Reply:
Ah yeah, I noted that fallthrough after sending it, so I deleted the comment because I didn't want to bother you. Sorry 😅

Reply:
No worries!